libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -256,6 +256,7 @@ void GCS_MAVLINK::send_battery_status(const uint8_t instance) const
     }
 
     float current, consumed_mah, consumed_wh;
+    int32_t time_remaining;
```

Comment:
Tighten the scope on this - we don't need to declare it until we go to fill it in.